my-react-app/src/app/screens/Login/index.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+import LoginForm from './layout.js';
+
+class LoginFormContainer extends Component {
+  submit = values => {
```

Comment:
The handler functions should be called `handleVerb`, and the prop `onVerb`. In this case, this should be called `handleSubmit` instead of `submit`.